lib/oplog/tail_test.go

```diff
@@ -142,7 +142,7 @@ func TestParseRawOplogEntry(t *testing.T) {
 				Operation: "u",
 				Namespace: "foo.Bar",
 				Doc:       mustRaw(t, map[string]interface{}{"new": "data"}),
-				Update:    rawOplogEntryID{ID: "updateid"},
+				Update:    rawBson(t, map[string]interface{}{"_id": "updateid"}),
```

Comment:
I see you used `rawBson` above and `mustRaw` in the other test cases. Why so? The pattern I notice seems to be that `mustRaw` is used for the input, and `rawBson` for the expected result. `mustRaw` seems weird actually. Other than what `rawBson` does, it checks if a byte array can be unmarshalled into `bson.Raw`, but that doesn't seem helpful, because they both have the same underlying type.

Reply:
Yeah, mustRaw was already there, and I added rawBson for the other tests, so when I had to update this I used the rawBson, but I see now they're basically the same so I replaced all the mustRaw with rawBson to simplify.